lib/ExtUtils/Liblist/Kid.pm

```diff
@@ -89,8 +84,15 @@ sub _unix_os2_ext {
                 elsif ( $Config{'lddlflags'} =~ /-R/ ) {
                     $rtype = '-R';
                 }
+                if ($thislib =~ s{(?<!\$)\$(?!\$)}{\\\$\$}g) {
```

Comment:
This change really ought to be a separate commit

Reply:
I had gone back and forth on that since letting $ORIGIN through on the rpaths was the point. I'll break it up.